hazelcast/src/test/java/com/hazelcast/cache/jsr/CacheEntryListenerClientServerTest.java

```diff
@@ -19,23 +19,18 @@
 import com.hazelcast.test.HazelcastSerialClassRunner;
 import com.hazelcast.test.annotation.QuickTest;
 import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
 @RunWith(HazelcastSerialClassRunner.class)
 @Category(QuickTest.class)
-public class CacheEntryListenerClientServerTest
-        extends org.jsr107.tck.event.CacheEntryListenerClientServerTest {
+public class CacheEntryListenerClientServerTest extends org.jsr107.tck.event.CacheEntryListenerClientServerTest {
 
-    /*
-    These system propreties are necessary if you run jsr tests from IDEA
```

Comment:
Who is supposed to find hints like this in only one of the test classes?

Reply:
No one!